static/configs/tslint_json

```diff
@@ -0,0 +1,129 @@
+            true,
+            "check-space"
+        ],
+        "curly": true,
+        "eofline": true,
+        "forin": true,
+        "import-spacing": true,
+        "indent": [
+            true,
+            "spaces",
+            4
+        ],
+        "interface-name": [
+            true,
+            "never-prefix"
+        ],
+        "interface-over-type-literal": true,
+        "label-position": true,
+        "max-line-length": [
+            true,
+            {
+                "limit": 140,
+                "ignore-pattern": "^import |^export {(.*?)}"
+            }
+        ],
+        "member-access": false,
+        "member-ordering": [
+            true,
+            {
+                "order": [
+                    "static-field",
+                    "instance-field",
+                    "static-method",
+                    "instance-method"
+                ]
+            }
+        ],
+        "no-arg": true,
+        "no-bitwise": true,
+        "no-console": [
+            true,
+            "debug",
+            "info",
+            "time",
+            "timeEnd",
+            "trace"
+        ],
+        "no-construct": true,
+        "no-debugger": true,
+        "no-duplicate-variable": true,
+        "no-empty": false,
+        "no-empty-interface": true,
+        "no-eval": true,
+        "no-inferrable-types": true,
+        "no-shadowed-variable": true,
+        "no-string-literal": false,
+        "no-string-throw": true,
+        "no-switch-case-fall-through": true,
+        "no-trailing-whitespace": true,
+        "no-unused-expression": true,
+        "no-var-keyword": true,
+        "object-literal-shorthand": [
+            true,
+            "never"
+        ],
+        "object-literal-sort-keys": false,
+        "no-floating-promises": true,
+        "one-line": [
+            true,
+            "check-open-brace",
+            "check-catch",
+            "check-whitespace"
+        ],
+        "prefer-const": false,
+        "quotemark": [
+            false,
+            "single"
+        ],
+        "radix": true,
+        "semicolon": [
+            true,
+            "always"
+        ],
+        "triple-equals": [
+            true,
+            "allow-null-check"
+        ],
+        "typedef-whitespace": [
+            true,
+            {
+                "call-signature": "nospace",
+                "index-signature": "nospace",
+                "parameter": "nospace",
+                "property-declaration": "nospace",
+                "variable-declaration": "nospace"
+            }
+        ],
+        "unified-signatures": true,
+        "variable-name": [
+            true,
+            "check-format",
+            "allow-snake-case",
+            "require-const-for-all-caps"
+        ],
+        "whitespace": [
+            true,
+            "check-branch",
+            "check-decl",
+            "check-operator",
+            "check-separator",
+            "check-type"
```

Comment:
I always feel confused about it. For example:

`"check-branch" checks branching statements (if/else/for/while) are followed by whitespace.` ([link](https://palantir.github.io/tslint/rules/whitespace/))

Is the rule enforcing adding or removing a space after the ìf`?

Reply:
Enforces adding. I'm attaching pictures for a demo. Also, notice the "whitespace" rule goes hand-in-hand with the "one-line".

<img width="406" alt="Captura de pantalla 2021-01-13 a las 16 08 23" src="https://user-images.githubusercontent.com/2401492/104470368-513e8400-55c2-11eb-94a8-e42a97605713.png">
<img width="486" alt="Captura de pantalla 2021-01-13 a las 16 08 42" src="https://user-images.githubusercontent.com/2401492/104470400-58fe2880-55c2-11eb-932e-ff08f5c67e62.png">
<img width="189" alt="Captura de pantalla 2021-01-13 a las 16 08 54" src="https://user-images.githubusercontent.com/2401492/104470436-61eefa00-55c2-11eb-90af-adb9bd998db1.png">

Reply:
In this case, I would suggest adding one more rule:
"check-module" checks for whitespace in import & export statements.

Reply:
`check-module` added in: https://github.com/mobilejazz/harmony-reference/pull/82/commits/2da646e1f0fb00c233eef773262f54b66eaeccd0